src/libs/address.js

```diff
@@ -62,12 +73,13 @@ export function normalize(type, raw) {
     };
   }
 
-  if (type === 'idunn') {
+  if (type === 'idunn' || type === 'latlon') {
```

Comment:
Here `type` describes the API where the `raw` object comes from : Idunn (the main API for all places), or Bragi (the geocoder used for search). So this extra condition should not be necessary.